README.md

```diff
@@ -100,6 +102,17 @@ Any new benchmarks must be named `*_benchmark.dart` and reside in the
 - [ ] shortestPath
 - [ ] unkinkPolygon
 
+### HELPER
```

Comment:
We have to decide if we need those helpers, because in Dart we have those amazing constructors, that JS/TS doesn’t have!